### PR TITLE
docs: add commit message format guidelines to the contributing guide

### LINKED ISSUE #5: Git verb for pull-request
__QUESTION__: What is the list with use cases?

__QUESTION__: Should we also add them in issues or commit names or only in the title of the pull-request?

### PR DESCRIPTION
**Pull request by hilnius**
Define our commit message format guidelines with a list of allowed commit verbs.

Resolves #5

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,3 +49,33 @@ Complete the form.
 4. Apply form
 
 ![alt text](images/create_pull_request.png "txt test")
```
Comment (slashome) [inline CONTRIBUTING.md]: txt test ? :)

Comment (slashome) [inline CONTRIBUTING.md]: I don't get why there is the 4 numbered bubble as there is not mention of these point in the documentation below

Comment (hilnius) [reply CONTRIBUTING.md]: not from my commit

Comment (hilnius) [reply CONTRIBUTING.md]: not from my commit too

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -49,3 +49,33 @@ Complete the form.
...
+
+Each commit message has a **type**, an optional **scope** and a **subject**:
+
+```
+<type>(<scope>): <subject>
+```
+
+Sample
+```
+docs: add commit message format guidelines to the contributing guide
```
Comment (slashome) [inline CONTRIBUTING.md]: the sample does not provide example for scope

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -49,3 +49,33 @@ Complete the form.
...
+```
+
+### Type
+Must be one of the following:
+
+* **build**: Changes that affect the build system or external dependencies
+* **ci**: Changes to our CI configuration files and scripts
+* **docs**: Documentation only changes
+* **feat**: A new feature
```
Comment (slashome) [inline CONTRIBUTING.md]: no more sub-feat ? does that mean that every ticket will be a feat PR with a [no-releaselog] flag ?

Comment (hilnius) [reply CONTRIBUTING.md]: I think so (just like all sub-feat have a [no-releaselog]) 
but I think we should follow the rule "if a commit adds a feature, it should be named 'feat'" as long as it's technically a new feature (not what the product would call a feature)

File: CONTRIBUTING.md (hunk 2 above)
Comment (hilnius) [reply CONTRIBUTING.md]: I was wondering if we should even keep the "scope" param, because I'm not sure what we would say as a list of valid scopes...

Comment (slashome) [reply CONTRIBUTING.md]: sure I think we can remove it for now

File: CONTRIBUTING.md (hunk 3 above)
Comment (slashome) [reply CONTRIBUTING.md]: so if it's not adding a feat ? for instance just a route, what will it be ?

Comment (hilnius) [reply CONTRIBUTING.md]: a new route is a feature, isn't it ?

Comment (slashome) [reply CONTRIBUTING.md]: It is but every PR is not I think but we can change it later in case we need sub-feat

Comment (hilnius) [reply CONTRIBUTING.md]: can you give be an example of a PR that does not match the feature description for you and should deserve a "sub-feat" label ?

Comment (slashome) [reply CONTRIBUTING.md]: A front component creation ?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -49,3 +49,33 @@ Complete the form.
 4. Apply form
 
 ![alt text](images/create_pull_request.png "txt test")
+
+## Commit Message Guidelines
+
+We define how to name and format our commits in order to have a clean **project history**, because it is more readable and because wee use the commit messages to **generate our release log**.
```
Comment (djslimbo) [inline CONTRIBUTING.md]: 'wee' should be 'we'

**Pull request merged**